buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_26_0.groovy

```diff
@@ -27,7 +27,7 @@ class GrpcVendoring_1_26_0 {
   static def protobuf_version = "3.11.0"
   static def grpc_version = "1.26.0"
   static def gson_version = "2.8.6"
-  static def netty_version = "4.1.42.Final"
+  static def netty_version = "4.1.50.Final"
```

Comment:
We should update the version of gRPC we are vendoring instead of changing the versions we are using here.

Reply:
Updated the grpc version.

Reply:
Here is an example of an update: https://github.com/apache/beam/pull/10578 (the process was different back then but breaks down to):
1) Create a new GrpcVendoring_1_31_0.groovy file with the updated deps
2) Release the vendored artifact (https://s.apache.org/beam-release-vendored-artifacts)
3) Consume the newly released artifact by updating all the java import paths etc..

It would be best to separate the gRPC update process from this PR.